### ISSUE TITLE
vfs: use thumbnail for files 

### ISSUE THREAD

**Issue by PaulLereverend** [owncloud/client#7156]
Hello,

I don't know if it is possible but it could be nice if the virtual files could have a thumbnail like a normal file would (word icon, pdf icon or image thumbnail) instead of the current blank one. It will be easier to distinguish especially when the files are not clearly named. 

### Expected behaviour

Files icon depending of the file type

### Actual behaviour

Blank icon on every file 

### Steps to reproduce
1. Install client
2. Enable virtual file system
3. Sync some files

**Comment by guruz** [comment]
This is definitely something we have thought about too.
If you're a Windows user, please check the latets 2.6.0 alpha1 which has a more native VFS integration in Explorer
https://github.com/owncloud/client/releases

**Comment by PaulLereverend** [comment]
It was exactly what I had in mind. Do you think it will be possible to bring it to MacOS and Linux too ?

**Comment by michaelstingl** [comment]
> Do you think it will be possible to bring it to MacOS and Linux too ?

Sure, but it will be a longer journey…

**Comment by lulle2007200** [comment]
Im at 2.6.0 alpha 1 and im not sure, what i am supposed to expect. I dont get thumbnails for images (neither for hydrated nor for non hydrated files, see image).
![image](https://user-images.githubusercontent.com/13625968/59153080-ea560200-8a51-11e9-8cf8-939ca1786bb0.png)

**Comment by ckamm** [comment]
Relates to #7175. Before we can have a thumbnail for dehydrated files we need to forward the thumbnail for hydrated files.

**Comment by TheOneRing** [comment]
Duplicate of 7175

**Comment by ngvoicu** [comment]
Image thumbnails for virtual files are not showing. Is there any progress on this matter?
